tests/integration-tests/RtpsDurableReplay/run_test.pl

```diff
@@ -0,0 +1,30 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+    & eval 'exec perl -S $0 $argv:q'
+    if 0;
+
+# -*- perl -*-
+
+use Env (DDS_ROOT);
+use lib "$DDS_ROOT/bin";
+use Env (ACE_ROOT);
+use lib "$ACE_ROOT/bin";
+use PerlDDS::Run_Test;
+use File::Path;
+use strict;
+
+PerlDDS::add_lib_path('../ConsolidatedMessengerIdl');
```

Comment:
```suggestion
PerlDDS::add_lib_path('../../DCPS/ConsolidatedMessengerIdl');
```
Or move this test's directory into DCPS.  It does use the DCPS API just like most of the other tests there.

Reply:
Oh I didn't notice this wasn't in `DCPS`. It'd be great if we could rename `tests/DCPS` to `integration-tests`, since that's what those tests basically are. However I think this should just be moved there for now and we could rename the directory all at once another time.